packages/common/styles/_tokens.scss

```diff
@@ -0,0 +1,8 @@
+@use './colors';
+
+// Font tokens
+$dsco-font-color: colors.$charcoal;
+$dsco-font-size: 14px;
+
+// Link tokens
+$dsco-link-color: colors.$teal;
```

Comment:
note: naming convention is still somewhat up-in-the air, but there will be a hierarchy to token names similar to what's here